osgbuild/main.py

```diff
@@ -314,7 +314,7 @@ def parse_cmdline_args(argv):
     parser = OptionParser(header)
     parser.add_option(
         "-a", "--autoclean", action="store_true",
-        help="Clean out the following directories before each build: "
+        help="(default) Clean out the following directories before each build: "
```

Comment:
This approach to indicating a default-on option seems to go against convention. What I am seeing in a random sampling of man pages is prose, typically at the end of the description. So perhaps something like “This option is on by default.” Whatever you do here should be done throughout, with changes to the individual items as needed.